Sources/FaunaDB/QueryResult.swift

```diff
@@ -70,6 +95,41 @@ extension QueryResult {
 
 extension QueryResult {
 
+    /**
+        Apply a transformation if an error has occurred during the query execution.
+
+        If `mapErr` returns a value, the resulting `QueryResult` will be considered a success.
```

Comment:
This is fine but I kinda feel that considered is a little too weak of a word when referring to software behavior. Maybe  "transformed into a success". instead